widgets/Snowflake-batchsource.json

```diff
@@ -200,6 +227,30 @@
         }
       ]
     },
+    {
+      "name": "ImportQuery",
+      "condition": {
+        "expression": "importQueryType == 'importQuery'"
```

Comment:
make this condition != 'tableName', in that case it will work for null in case of upgrade